Stripe/PublicHeaders/STPPaymentMethodParams.h

```diff
@@ -184,6 +190,17 @@ NS_ASSUME_NONNULL_BEGIN
                                         billingDetails:(STPPaymentMethodBillingDetails *)billingDetails
                                               metadata:(nullable NSDictionary<NSString *, NSString *> *)metadata;
 
+/**
+ Creates params for a giropay PaymentMethod;
+
+ @param eps   An object containing additional EPS details.
+ @param billingDetails  An object containing the user's billing details. Note that `billingDetails.name` is required for giropay PaymentMethods.
```

Comment:
giropay -> EPS

Also, is the Note true?  Is billingDetails.name the single required field?

Reply:
https://stripe.com/docs/sources/eps#create-source

Note's correct